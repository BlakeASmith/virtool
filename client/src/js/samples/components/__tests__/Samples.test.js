import Samples, { SampleFileManager, SampleSettings } from "../Samples";

describe("<SampleFileManager />", () => {
    it("should render", () => {
        const wrapper = shallow(<SampleFileManager />);
        expect(wrapper).toMatchSnapshot();
    });
});

describe("<SamplesSettings />", () => {
    it("should render", () => {
        const wrapper = shallow(<SampleSettings />);
        expect(wrapper).toMatchSnapshot();
    });
});

describe("<Samples />", () => {
    it("should render", () => {
        const wrapper = shallow(<Samples />);
        expect(wrapper).toMatchSnapshot();
    });
});
