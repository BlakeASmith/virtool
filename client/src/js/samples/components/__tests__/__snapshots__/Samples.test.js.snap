// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SampleFileManager /> should render 1`] = `
<Connect(FileManager)
  fileType="reads"
/>
`;

exports[`<Samples /> should render 1`] = `
<div
  className="container"
>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(SamplesList)",
          "type": [Function],
        }
      }
      exact={true}
      path="/samples"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/samples/files"
    />
    <Route
      component={[Function]}
      path="/samples/settings"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(SampleDetail)",
          "type": [Function],
        }
      }
      path="/samples/:sampleId"
    />
  </Switch>
</div>
`;

exports[`<SamplesSettings /> should render 1`] = `
<div
  className="settings-container"
>
  <h3
    className="view-header"
  >
    <strong>
      Sample Settings
    </strong>
  </h3>
  <Connect(UniqueNames) />
  <Connect(SampleRights) />
</div>
`;
